MonumentAddons.cs

```diff
@@ -5197,6 +5197,14 @@ public override void ObjectSpawned(SpawnPointInstance instance)
                     SpawnedVehicleComponent.AddToVehicle(Adapter.PluginInstance, instance.gameObject);
                     entity.Invoke(() => DisableVehicleDecay(entity), 5);
                 }
+
+                var hackableCrate = entity as HackableLockedCrate;
+                if ((object)hackableCrate != null && hackableCrate.shouldDecay)
+                {
+                    hackableCrate.shouldDecay = false;
+                    hackableCrate.CancelInvoke(hackableCrate.DelayedDestroy);
+                    hackableCrate.InvokeRepeating(() => SetHackableCrateDecay(hackableCrate), UnityEngine.Random.Range(0f, 5f), 300f);
```

Comment:
This invoke should probably be added to the custom component instead of to the entity itself. It doesn't matter right now, but if we ever support saving entities spawned by spawn points, attaching the invoke to the component would provide for free the ability to remove and replace the invoke.

Reply:
I dont understand this one. Do you mean it like `hackableCrate.GetOrAddComponent<SomeComponent>().InvokeRepeating`? I cant see the difference

Reply:
I was thinking on the `SpawnPointInstance` component since those will exist on all the objects spawned by spawn groups.